include/liblightnvm.h

```diff
@@ -1111,6 +1111,13 @@ void nvm_addr_pr(const struct nvm_addr addr);
 void nvm_addr_prn(const struct nvm_addr *addr, unsigned int naddrs,
 		  const struct nvm_dev *dev);
 
+/**
+ * Prints a terse output for the given list of addresses
+ * according to the geometry of the given device.
+ */
+void nvm_addr_terse_prn(const struct nvm_addr *addr, unsigned int naddrs,
+                const struct nvm_dev *dev);
+
 /**
```

Comment:
I'd prefer that we leave this function out, if you use the existing `nvm_addr_pr` for the other changes then this function is only used once.

Reply:
Removed it.